builds/heroku-18.Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM heroku/heroku:18-build
 WORKDIR /app
 ENV WORKSPACE_DIR="/app/builds" \
     S3_BUCKET="lang-python" \
-    S3_PREFIX="heroku-18/"
+    S3_PREFIX="heroku-18/" \
+    STACK="heroku-18"
```

Comment:
Can you explain what the stack var is for @dzuelke? is this so we don't have to specify it in the test steps like in the Makefile?

Reply:
It's mostly so that it's available to formulae in case they have to do different things depending on the stack (say `apt-get install …-dev` packages)